pkgs/by-name/ta/talecast/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "talecast";
+  version = "0.1.39";
+
+  src = fetchFromGitHub {
+    owner = "TBS1996";
+    repo = pname;
+    rev = "816e11dcc5eafbe54d2ecd4bc525a46f88a3e154";
+    hash = "sha256-sR0wyzAwTTwO+HHly4I0JJ5HwxJUD1yGdKl9LqEbnI0=";
+  };
+
+  cargoHash = "sha256-mIzrYlAqHYrK2bb/ZUzqIwhPJKcTQpNpqijpEuwLc5A=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl ];
+
```

```suggestion
  passthru = {
    updateScript = nix-update-script { };
    tests.version = testers.testVersion { package = talecast; };
  };

```
we can adding basic tests and an update script here to make this easier to maintain